src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -185,15 +192,14 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
 
     public async getPathToFile(fileName: string, directory: string) {
         const dirList = directory.split("/"); // Hardcoded as provisioning service is running on linux and we cannot use Path.sep as it is machine dependent
-        let directoryPath: string = "";
-        directoryPath = await this.localGitRepoHelper.getGitRootDirectory();
+        let directoryPath: string = this.tmpDirectoryPath;
         dirList.forEach((dir) => {
             try {
                 directoryPath = Path.join(directoryPath, dir);
                 // tslint:disable-next-line:non-literal-fs-path
-                if (!fs.existsSync(directoryPath)) {
+                if (!fse.existsSync(directoryPath)) {
                     // tslint:disable-next-line:non-literal-fs-path
-                    fs.mkdirSync(directoryPath);
+                    fse.mkdirSync(directoryPath);
```

Comment:
Not related to your change, but is there no flavor of mkdir which creates all intermediate directories? Then we won't need to create all of them in a loop.

Reply:
Yes we do have ``mkdirpSync`` in fs-extra module, but we need keep this loop because path is coming from provisioning service (which is on unix) and we need to create path based on user os so we cannot pass the same path to ``mkdirpSync`` function